engine/src/main/java/org/pentaho/di/trans/steps/fileinput/text/TextFileInputUtils.java

```diff
@@ -291,10 +291,10 @@ public static final String getLine( LogChannelInterface log, InputStreamReader r
   }
 
   public static final String getLine( LogChannelInterface log, InputStreamReader reader, EncodingType encodingType,
-                                      int fileFormatType, StringBuilder line, String regex )
+                                      int fileFormatType, StringBuilder line, String regex, String escapeChar )
```

Comment:
As these methods are public in an "Util" class, it would be better not to change its signature.
Leave these new versions as is and add the former ones again where the only thing they do is to invoke the new versions with the proper "escapeChar" string to maintain the behaviour.